libs/analytics/src/pixels/initPixelAnalytics.ts

```diff
@@ -0,0 +1,26 @@
+import { isEns, isProd } from '@cowprotocol/common-utils'
+
+import { PixelAnalytics } from './PixelAnalytics'
+import { FacebookPixel } from './providers/facebook'
+import { LinkedinPixel } from './providers/linkedin'
+import { MicrosoftPixel } from './providers/microsoft'
+import { PavedPixel } from './providers/paved'
+import { RedditPixel } from './providers/reddit'
+import { TwitterPixel } from './providers/twitter'
+
+
+export function initPixelAnalytics(): PixelAnalytics | undefined {
+  const isPixelEnabled = isProd || isEns
+  if (!isPixelEnabled) {
+    return undefined
+  }
+
+  return new PixelAnalytics([
+    new FacebookPixel(),
+    new LinkedinPixel(),
+    new MicrosoftPixel(),
+    new PavedPixel(),
+    new RedditPixel(),
+    new TwitterPixel(),
```

Comment:
Does this mean all of those are on by default on prod now?

Reply:
Yes, good point.

how it works now is that it relies on the factory method to decide if this is needed or not.

```
if (!isPixelEnabled) {
    return undefined
  }
```

And this is currently using the same logic as before:
https://github.com/cowprotocol/cowswap/blob/e2fbecba5ec1c0b49316643d8ddf1fb443b187b3/libs/analytics/src/pixel/constants.ts#L3

replaced with:

```ts
const isPixelEnabled = isProd || isEns
```

However, its a good point, because before it was enable on prod, but it had no effect because we commented out the pixels https://github.com/cowprotocol/cowswap/blob/e2fbecba5ec1c0b49316643d8ddf1fb443b187b3/apps/cowswap-frontend/src/common/hooks/useAnalyticsReporter.ts#L104

What I will do is to make it return undefined always :P

Reply:
6b75e6f37e5b0eda69c2cff6a5805dc29027839d